### PR TITLE
QNAP TS-251+

### PR DESCRIPTION
**Pull request by madecto**

### CODE REVIEW

**Comment by madecto** [comment]
Goodday @danner26, I'm struggling to get this PR work: it appears new enforcements are in place for device `slug` and filenames and not on `model` and `part_number`: as this Model and Part Number both include the _forbidden_ character "+" the verification fails.
I could swap the "plus" string to the "+" sign, but when I look up a device type I would personally much prefer to find the vendor grammar rather than an alternative version.

**Review by abhi1693 — CHANGES_REQUESTED** [review]
Hello, it appears that you have modified the `tests/known-slugs.pickle` file. This file is automatically generated via a GitHub Action, which contains all currently available slugs. This file should **never** be commited by a contributor in a PR.
  
In order to fix this, you will need to remove the modification on the pickle file. There are a handful of ways to do so but often it is easiest to revert the commit and force push it again without the pickle file included. If you need assistance you are welcome to ping a maintainer via this PR, or via our NetDev Slack.

File: device-types/QNAP/TS-251-plus.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+manufacturer: QNAP
+model: TS-251+
+slug: qnap-ts-251-plus
+part_number: TS-251+
+is_full_depth: false
+weight: 1.28
```
Comment (danner26) [inline device-types/QNAP/TS-251-plus.yml:7]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/QNAP/TS-251-plus.yml (hunk 1 above)
Comment (madecto) [reply device-types/QNAP/TS-251-plus.yml:7]: Hello @danner26,
I've managed to grab this from a real-life example.

Comment (madecto) [reply device-types/QNAP/TS-251-plus.yml:7]: Hello @danner26,
I've managed to grab this from a real-life example.

**Pull request merged**